### PR TITLE
Site search improvements

### PR DESCRIPTION
**Pull request by yannyu**
Minor fixes to query pipelines and *:* queries

### CODE REVIEW

File: python/fusion_config/query_default_pipeline.json [hunk 1]
```diff
@@ -56,7 +61,7 @@
         },
         {
           "key": "fl",
-          "value": "id title productVersion productName og_image parent_s",
+          "value": "id title productVersion productName og_image parent_s author project_label datasource_label",
```
Comment (gsingers) [inline python/fusion_config/query_default_pipeline.json]: @yannyu why the two different "fl" settings.  The "Query Fields" stage has a "return fields" setting.  Add them there.

File: python/fusion_config/query_site_search_all_pipeline.json [hunk 2]
```diff
@@ -2,6 +2,18 @@
...
+    "id" : "j90y8a5jbijurf6r",
+    "params" : [ {
+      "key" : "fl",
+      "value" : "title content body subject accepted_answer_txt answer_txt",
+      "policy" : "append"
+    } ],
+    "type" : "set-params",
+    "skip" : false,
+    "label" : "set-params",
+    "condition" : "request.getFirstParam(\"q\") == \"*\" || request.getFirstParam(\"q\") == \"*:*\""
```
Comment (gsingers) [inline python/fusion_config/query_site_search_all_pipeline.json:40]: How's the performance of this look?

File: python/fusion_config/query_site_search_blog_pipeline.json [hunk 3]
```diff
@@ -39,7 +51,24 @@
...
+    }, {
+      "key": "bq",
+      "value": "(productVersion:3.0)^10",
+      "policy": "append"
+    }, {
+      "key": "bq",
+      "value": "(productVersion:2.4)^5",
+      "policy": "append"
+    }, {
+      "key": "mm.autoRelax",
```
Comment (gsingers) [inline python/fusion_config/query_site_search_blog_pipeline.json]: We need to set the `mm` value on the Query Fields Stage (scroll to the bottom)

File: python/fusion_config/query_site_search_documentation_pipeline.json [hunk 4]
```diff
@@ -73,6 +85,11 @@
         "key": "bq",
         "value": "productVersion:2.4^5",
         "policy": "append"
+      },
+      {
+        "key": "mm.autoRelax",
+        "value": "true",
```
Comment (gsingers) [inline python/fusion_config/query_site_search_documentation_pipeline.json]: Same about the `mm` setting.

File: python/fusion_config/query_site_search_videos_pipeline.json [hunk 5]
```diff
@@ -39,7 +51,24 @@
...
+    }, {
+      "key": "bq",
+      "value": "(productVersion:3.0)^10",
+      "policy": "append"
+    }, {
+      "key": "bq",
+      "value": "(productVersion:2.4)^5",
+      "policy": "append"
+    }, {
+      "key": "mm.autoRelax",
```
Comment (gsingers) [inline python/fusion_config/query_site_search_videos_pipeline.json]: Same about mm

**Review by gsingers — CHANGES_REQUESTED** [review]
Overall looks good.  Check again for consistency across a few places.

File: python/fusion_config/query_site_search_all_pipeline.json (hunk 2 above)
Comment (yannyu) [reply python/fusion_config/query_site_search_all_pipeline.json:40]: It's only the * query for each page, so it's not bad and caches well.

File: python/fusion_config/query_default_pipeline.json [hunk 6]
```diff
@@ -81,7 +81,7 @@
           "policy" : "append"
         }, {
           "key" : "hl.fl",
-          "value" : "title content body subject question_txt",
+          "value" : "title content body subject accepted_answer_txt answer_txt",
```
Comment (gsingers) [inline python/fusion_config/query_default_pipeline.json:84]: We prob. should highlight in the ?_text too?

File: python/fusion_config/query_site_search_all_pipeline.json [hunk 7]
```diff
@@ -1,193 +1,52 @@
...
+    }, {
+      "key" : "group.limit",
+      "value" : "10",
+      "policy" : "append"
+    } ],
+    "type" : "set-params",
+    "skip" : false,
+    "label" : "QueryParams - Grouping"
+  }, {
+    "type" : "set-params",
```
Comment (gsingers) [inline python/fusion_config/query_site_search_all_pipeline.json:30]: Can this be a Query Fields Stage?

File: python/fusion_config/query_site_search_blog_pipeline.json [hunk 8]
```diff
@@ -1,76 +1,36 @@
...
-      "value" : "title^3 content productName productVersion",
+      "value" : "site_search_s:blog",
       "policy" : "append"
-    }, {
+    } ],
+    "type" : "set-params",
+    "skip" : false,
+    "label" : "QueryParams - FilterQuery"
+  }, {
+    "type" : "set-params",
```
Comment (gsingers) [inline python/fusion_config/query_site_search_blog_pipeline.json:22]: Same as above, can this be a Query Fields Stage?

File: python/fusion_config/query_site_search_documentation_pipeline.json [hunk 9]
```diff
@@ -17,95 +35,17 @@
...
-      "value" : "id title productVersion productName og_image",
-      "policy" : "append"
-    }, {
-      "key" : "hl",
-      "value" : "true",
-      "policy" : "append"
-    }, {
-      "key" : "hl.fragsize",
-      "value" : "300",
+      "value" : "title content body subject accepted_answer_txt answer_txt",
```
Comment (gsingers) [inline python/fusion_config/query_site_search_documentation_pipeline.json:41]: Again.

File: python/fusion_config/query_site_search_videos_pipeline.json [hunk 10]
```diff
@@ -1,48 +1,35 @@
...
-      "value" : "300",
-      "policy" : "append"
-    }, {
-      "key" : "hl.snippets",
-      "value" : "2",
-      "policy" : "append"
-    }, {
-      "key" : "hl.fl",
-      "value" : "title content",
+      "value" : "title content body subject accepted_answer_txt answer_txt",
```
Comment (gsingers) [inline python/fusion_config/query_site_search_videos_pipeline.json:26]: here too

File: python/fusion_config/query_site_search_all_pipeline.json (hunk 7 above)
Comment (yannyu) [reply python/fusion_config/query_site_search_all_pipeline.json:30]: The Query Fields stage lets you put in qf and fl params, not sure how we'd shoehorn in these grouping params.

File: python/fusion_config/query_site_search_blog_pipeline.json (hunk 8 above)
Comment (yannyu) [reply python/fusion_config/query_site_search_blog_pipeline.json:22]: Similarly, this is declaring fq params, not qf or fl.

File: python/fusion_config/query_site_search_documentation_pipeline.json (hunk 9 above)
Comment (yannyu) [reply python/fusion_config/query_site_search_documentation_pipeline.json:41]: And here we're declaring hl and hl.fl params, not standard fl params or qf params.

**Pull request merged**